website/docs/docs/deploy/job-commands.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Job commands"
+id: "job-commands"
+description: "How to use dbt commands to set tasks for your dbt Cloud jobs."
+---
+
+A dbt Cloud production job allows you to set up a system to run a dbt job and job commands on a schedule, rather than running dbt commands manually from the command line or [IDE](/docs/get-started/develop-in-the-cloud). A job consists of multiple commands, and their combined outcomes determine the job's run status, which can be successful, canceled, or failed. During a job run, each command outcome can either succeed or fail under the **Run steps** section.
+
+Each job allows you to:
+
+- Configure job commands
+- View job run details, including timing, artifacts, and detailed run steps
+- Access logs to view or help debug issues and historical invocations of dbt
+- Set up notifications, and [more](/docs/deploy/deployments#dbt-cloud)!
+
+## Job command types
+
+Job commands are specific tasks executed by the job, and you can configure them seamlessly by either adding [dbt commands](/reference/dbt-commands) or using the checkbox option in the **Commands** section. 
+
+During a job run, the commands are "chained" together, and you can expect different outcomes when adding the command as a run step compared to selecting a checkbox.
+
+<Lightbox src ="/img/docs/dbt-cloud/using-dbt-cloud/job-commands.gif" title="Configuring checkbox and commands list"/>
+
+
+### Built-in commands
+
+Every job invocation automatically includes [`dbt deps`](/reference/commands/deps), meaning you don't need to add it to the **Commands** list in your job settings. 
+
+**Job outcome** &mdash; During a job run, the commands are "chained" together.  This means if one of the run steps in the chain fails, then the next commands aren't executed, and the entire job fails with an "Error" job status.
+
+
+<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/fail-dbtdeps.jpg" title="A failed job that had an error during the dbt deps run step."/>
+
+### Checkbox commands
+
+For every job, you have the option to select the [Generate docs on run](/docs/collaborate/build-and-view-your-docs) or [Run source freshness](/docs/deploy/source-freshness) checkboxes. These checkboxes enable you to generate project docs or enable source freshness automatically.
+
+**Job outcome**  &mdash; During a job run, if either "checkbox command" (source freshness or generate documentation) fails, the job can still be successful if all subsequent steps are successful. 
```

Comment:
This likely needs to be split out to describe the behavior for source freshness and docs, mostly because of the sequencing:

- checkbox source freshness: runs before manual commands, non blocking of subsequent commands regardless of status
- checkbox docs: runs _after_ manually specified commands, non blocking of those commands as a result of the timing. I am fairly sure that a failure on the docs generate step will produce a job status of failure as well, so that's worth clarifying

Reply:
thanks so much Dave, i've asked in dbt help and tagged you too! the docs say it wont fail so i don't know!